### ISSUE TITLE
--quick events created with 4.0.0alpha no longer honor calendar default notification

### ISSUE THREAD

**Issue by ronjouch** [insanum/gcalcli#327]
Use case: `gcalcli --calendar 'My Calendar' quick '5pm Feed brontosaurus'`

- With 3.4.0, the newly-created event used to have a notification associated to it (e.g. "30 min before"), inherited from my calendar's default notification.
- With 4.0.0a3, the newly-created event has no notification associated to it.

Is it a regression, or am I missing something?

**Comment by jcrowgey** [comment]
I think its a regression.

**Comment by svermeulen** [comment]
Yeah, no matter what I do I can't seem to get events created with `quick` to have any notifications.  I've tried both setting one explicitly using `--reminder` and also `--default-reminders` but both result in zero notifications set up

**Comment by svermeulen** [comment]
I tried 3.4.0 and had the same issue btw

**Comment by reagle** [comment]
Almost three years later, with gcalcli v4.3.0; this is still an issue?!
